{{cookiecutter.project_slug}}/compose/production/traefik/traefik.yml

```diff
@@ -5,6 +5,11 @@ entryPoints:
   web:
     # http
     address: ":80"
+    http:
+      redirections:
+        entryPoint:
+          to: web-secure
+          scheme: https
```

Comment:
Could we add a comment linking to the relevant piece of Traefik documentation?

Reply:
Made and pushed the change. Also I removed [`scheme: https`](https://docs.traefik.io/routing/entrypoints/#entrypoint) as it is the default value anyway